clusterloader2/testing/huge-service/config.yaml

```diff
@@ -0,0 +1,33 @@
+# Huge service test config
+{{$HUGE_SERVICE_HEADLESS := DefaultParam .CL2_HUGE_SERVICE_HEADLESS false}}
+{{$HUGE_SERVICE_ENDPOINTS := DefaultParam .CL2_HUGE_SERVICE_ENDPOINTS 1000}}
+
+name: huge-service
+namespace:
+  number: 1
+tuningSets:
+- name: Sequence
```

Comment:
We have three different tuningSets to remove single object :)

Please use single one everywhere (e.g. Sequence).

Reply:
Fixed.